src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Utf8JsonReaderTests.cs

```diff
@@ -3824,6 +3824,7 @@ static void Test(ref Utf8JsonReader reader)
                 Assert.True(reader.Read());
                 Assert.Equal(JsonTokenType.String, reader.TokenType);
                 Assert.True(reader.ValueIsEscaped);
+                Assert.Contains((byte)'\\', reader.HasValueSequence ? reader.ValueSequence.ToArray() : reader.ValueSpan.ToArray());
```

Comment:
can we check the entire sequence?

Reply:
We don't need to. That's the responsibility of other unit tests.